ipfs-cluster/test_tests/old_range.yml

```diff
@@ -0,0 +1,14 @@
+name: Test old Onstep Endstep
+config:
+  nodes: 5
+  selector: app=ipfs-cluster
+  times: 1
+  expected:
+      successes: 0
+      failures: 0
+      timeouts: 0
+steps:
+  - name: Run on old step nodes 2 3 4
+    cmd: echo "old range" > OLD_RANGE
+    on_node: 2
+    end_node: 4
```

Comment:
Thanks Hector, I realize that this is a hack.  As you said, I could not find a way to specifically identify which node was being called using assertions in the DSL.  

It turned out that the echoed output was actually less important than looking at the testing output.  I tested once to see that the indices passed in to handleStep are actually getting run, and then for the rest of the tests the real check is to see whether the expected nodes (in the test name) are the same as the actual nodes (output by default by the test framework).  Maybe it would be cleaner to make the command a noop of some kind?